public_html/wp-content/plugins/wordcamp-remote-css/app/synchronize-remote-css.php

```diff
@@ -77,7 +77,9 @@ function fetch_unsafe_remote_css( $remote_css_url ) {
  */
 function sanitize_unsafe_css( $unsafe_css ) {
 	if ( ! class_exists( 'Jetpack_Custom_CSS_Enhancements' ) ) {
-		if ( version_compare( JETPACK__VERSION, '11.6', '<' ) ) {
+		if ( version_compare( JETPACK__VERSION, '13.7.1', '>' ) ) {
+			return ''; // Custom CSS not included in Jetpack anymore.
```

Comment:
This is obviously a big TODO; to replace the Jetpack CSS sanitization dependency with our own bundled here.
This is needed, and was one of the reasons for this PR in the first places.

This change should be reverted from the PR really.